pdns/recursordist/settings/docs-new-preamble-in.rst

```diff
@@ -472,6 +472,57 @@ An example of an ``proxymappings`` entry, which is a sequence of `ProxyMapping`_
          - example.com
          - example.net
 
+ForwardingCatalogZone
+^^^^^^^^^^^^^^^^^^^^^
+As of version 5.2.0, a forwarding catalog zone entry is defined as:
+
+.. code-block:: yaml
+
+     zone: Name of catalog zone
+     notify_allowed: bool, default false
+     xfr:
+       addresses: [] Sequence of SocketAddress
+       zoneSizeHint: number, default not set
+       tsig:
+         name: string
+         algo: string
+         secret: base64string
+       refresh: number, default not set
+       maxReceivedMBytes: number default not set
+       localAddress: IP address, default not set
+       axfrTimeout: number, default 20
+     groups:
+     - name:
+       forwarders: [] Sequence of SocketAddress
+       recurse: bool, default false
+       notify: bool, default false
+
+An example of an ``forwarding_catalog_zones`` entry, which is a sequence of `ForwardingCatalogZone`_:
+
+.. code-block:: yaml
+
+   forwarding_catalog_zones:
+   - zone: 'forward.example'
+     xfr:
+       addresses: [128.66.1.2]
+     groups:
+      - forwarders: [192.168.178.1] # default forwarder
+      - name: mygroup
+        forwarders: [192.168.179.2] # forwarder for catalog zone members in mygroup
+        recurse: true
+        notify_allowed: true
+   - zone: 'forward2.example'
+     xfr:
+       addresses: [128.66.1.3]
+     groups:
+      - forwarders: [192.168.178.3] # only default forwarder for 2nd catalog zone
+
+:program:`Recursor` will transfer the catalog zone from te authoritative server using IXFR (falling back to AXFR if needed) and add forwarding clauses for all members of the catalog zone.
```

Comment:
typo here "te"

Reply:
if you think `the` is _likely_ to be misspelled as `te`, then you could add this to `line_forbidden.patterns`:
```
# Should probably be `the`
\bte\b
```